src/commands/cluster-shards.json

```diff
@@ -11,9 +11,6 @@
             "LOADING",
             "STALE"
         ],
-        "command_tips": [
-            "NONDETERMINISTIC_OUTPUT"
-        ],
         "reply_schema": {
```

Comment:
The replication offset is still effectively non-determinsitic, so I think we nee to leave this flag. It's not used within the engine to determine anything, so it should be OK to leave.